### PR TITLE
Add question about congresspeople names to the FAQ

### PR DESCRIPTION
**Pull request by cuducos**

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -179,6 +179,16 @@ <h4 class="panel-title"><a aria-controls="collapseSeventeen" aria-expanded="fals
               </div>
             </div>
           </div>
+          <div class="panel panel-default">
+            <div class="panel-heading" id="headingSeventeen" role="tab">
+                <h4 class="panel-title"><a aria-controls="collapseSeventeen" aria-expanded="false" class="collapsed" data-parent="#accordion" data-toggle="collapse" href="#collapseSeventeen" role="button">18. Vocês estão protegendo os deputados não divulgando o nome dos suspeitos?</a></h4>
+            </div>
+            <div aria-labelledby="headingSeventeen" class="panel-collapse collapse" id="collapseSeventeen" role="tabpanel">
```
Comment (jtemporal) [inline faq.html]: @cuducos need to change the id numbering here to match the question number ;)

**Review by jtemporal — CHANGES_REQUESTED** [review]

File: faq.html (hunk 1 above)
Comment (cuducos) [reply faq.html]: Thanks ; )
Done.

File: faq.html [hunk 2]
```diff
@@ -168,9 +168,9 @@ <h4 class="panel-title"><a aria-controls="collapseSixteen" aria-expanded="false"
           </div>
           <div class="panel panel-default">
             <div class="panel-heading" id="headingSeventeen" role="tab">
-                <h4 class="panel-title"><a aria-controls="collapseSeventeen" aria-expanded="false" class="collapsed" data-parent="#accordion" data-toggle="collapse" href="#collapseSeventeen" role="button">17. O que o nome Serenata de Amor significa?</a></h4>
+                <h4 class="panel-title"><a aria-controls="collapseSeventeen" aria-expanded="false" class="collapsed" data-parent="#accordion" data-toggle="collapse" href="#collapseEightteen" role="button">17. O que o nome Serenata de Amor significa?</a></h4>
```
Comment (jtemporal) [inline faq.html:171]: Here should be seventeen in the href tag

File: faq.html [hunk 3]
```diff
@@ -168,9 +168,9 @@ <h4 class="panel-title"><a aria-controls="collapseSixteen" aria-expanded="false"
...
             </div>
-            <div aria-labelledby="headingSeventeen" class="panel-collapse collapse" id="collapseSeventeen" role="tabpanel">
+            <div aria-labelledby="headingSeventeen" class="panel-collapse collapse" id="collapseEightteen" role="tabpanel">
               <div class="panel-body">
```
Comment (jtemporal) [inline faq.html:174]: Also here in the id tag should be an collapseSeventeen

**Review by jtemporal — CHANGES_REQUESTED** [review]
=D

**Comment by anaschwendler** [comment]
Already done.

**Pull request closed without merging**